doc/monitoring/index.rst

```diff
@@ -14,6 +14,7 @@ This chapter includes the following sections:
     :numbered: 0
 
     getting_started
+    when_use_cartridge
```

Comment:
This patch adds a new documentation page, but doesn't remove existing cartridge-related info from getting started. I think this patch should provide a full migration

Reply:
I completely agree. I suggest doing this towards the end rewiev. When with the page "Getting started (Cartridge application)" everything will be clear.